ARE/services/CIMCommunication/src/main/java/eu/asterics/mw/cimcommunication/CIMIdentifyPortController.java

```diff
@@ -121,9 +121,13 @@ public void closePortInternal() {
         threadRunning = false;
```

Comment:
Maybe we could speedup the cancellation by additionally sending 

Thread.interrupt() (https://docs.oracle.com/javase/tutorial/essential/concurrency/interrupt.html) or using a Future to cancel (https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/Future.html#cancel(boolean)) the respective Threads.

Reply:
ok ... feel free to try this ;)